version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.6.1-SNAPSHOT"
```

Comment:
can you update this to 0.8.0-SNAPSHOT? because 0.7.1-SNAPSHOT will be made from branch-0.7

Reply:
Done 👍